package.json

```diff
@@ -40,7 +40,10 @@
         "semantic-release": "^17.4.2",
         "ts-jest": "^26.5.5",
         "ts-node": "^9.1.1",
-        "typescript": "^4.2.3"
+        "typescript": "~4.5.3"
+    },
+    "peerDependencies": {
+        "typescript": "^4.5"
```

Comment:
We need some section in the readme which says that version 0.x are compatible with TS 4.2 (and 4.3/4.4 probably), and that 1.x is compatible with 4.5

Reply:
sure thing, will add some section with compatibility info to readme